### PR TITLE
CORE-565 Changed the default data directory to ~/.rnode.

### PR DESCRIPTION
**Pull request by dzajkowski**
## Overview
Using `/var/lib/rnode` as the default data directory is terrible UX, so `$HOME/.rnode` was proposed as a replacement.

### Does this PR relate to an RChain JIRA issue? 
https://rchain.atlassian.net/projects/CORE/issues/CORE-565

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### Notes
none.

### CODE REVIEW

File: node-client/README.md [hunk 1]
```diff
@@ -25,8 +25,7 @@ pip install -r requirements.txt
 solution:
 
 ```
-sudo mkdir -p /var/lib/rnode
-sudo chown $USER /var/lib/rnode
+sudo mkdir -p ~/.rnode
```
Comment (henrytill) [inline node-client/README.md]: definitely don't need `sudo` or `-p` this.

better question: will it not create the directory automagically now?  

see:
https://github.com/rchain/rchain/blob/64f3c20b13fd3a5ebd4950d0286d2c11b344eafe/rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala#L92

File: node/README.md [hunk 2]
```diff
@@ -144,14 +144,13 @@ Node exposes its API via gRPC services, which are exposed on `grpc-port`. To see
...
-Node needs to have read and write access to a folder called data directory. By default that folder is `/var/lib/rnode/`. User can control that value by providing `--data_dir` flag. 
+Node needs to have read and write access to a folder called data directory. By default that folder is `$HOME/.rnode`. User can control that value by providing `--data_dir` flag. 
 Regardless of which path on the file system you choose for the data directory, please remember that node needs to have read and write access to that folder.
 
 Below is an example of creating folder for default location and giving it ownership to the current user.
 
 ```
-sudo mkdir -p /var/lib/rnode
-sudo chown $USER /var/lib/rnode
+sudo mkdir -p ~/.rnode
```
Comment (henrytill) [inline node/README.md]: see above

File: node/rnode.service [hunk 3]
```diff
@@ -3,7 +3,7 @@ Description=RChain Node
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/rnode
+ExecStart=/usr/bin/rnode --data_dir=/var/lib/rnode
```
Comment (henrytill) [inline node/rnode.service]: This depends on the change in #1038 to work correctly.

File: node/src/main/scala/coop/rchain/node/conf.scala [hunk 4]
```diff
@@ -77,8 +77,8 @@ final case class Conf(arguments: Seq[String]) extends ScallopConf(arguments) {
     val host = opt[String](default = None, descr = "Hostname or IP of this node.")
 
     val data_dir = opt[Path](required = false,
-                             descr = "Path to data directory. Defaults to /var/lib/rnode",
-                             default = Some(Paths.get("/var/lib/rnode")))
+                             descr = "Path to data directory. Defaults to $HOME/.rnode",
+                             default = Some(Paths.get(System.getProperty("user.home"), ".rnode")))
```
Comment (lukasz-golebiewski) [inline node/src/main/scala/coop/rchain/node/conf.scala]: `sys.props` is Scala idiomatic

File: node/README.md [hunk 5]
```diff
@@ -80,7 +80,7 @@ Subcommand: run
   -c, --certificate  <arg>      Path to node's X.509 certificate file, that is
                                 being used for identification
   -d, --data_dir  <arg>         Path to data directory. Defaults to
-                                /var/lib/rnode
+                                $HOME/.rnode
```
Comment (goral09) [inline node/README.md]: If I am correct I see that the docker is run with `--data_dir=/var/lib/rnode` . This is different right?

Comment (goral09) [reply node/README.md]: Ok, thanks :+1:

**Pull request merged**